src/seedu/addressbook/AddressBook.java

```diff
@@ -430,6 +441,27 @@ private static String executeAddPerson(String commandArgs) {
         return getMessageForSuccessfulAddPerson(personToAdd);
     }
 
+
+    private static String executeEditPerson(String commandArgs) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.